src/tail/tail.rs

```diff
@@ -346,7 +346,18 @@ fn follow<T: Read>(readers: &mut [BufReader<T>], filenames: &[String], settings:
     let mut read_some = false;
     let mut process = platform::ProcessChecker::new(settings.pid);
 
+    let mut buffer = BufReader::new(stdin());
     loop {
+        loop {
+            let mut datum = String::new();
+            match buffer.read_line(&mut datum) {
+                Ok(0) => break,
+                Ok(_) => {
+                  print!("input consumed: {}", datum);
```

Comment:
I would prefer to silently consume input, but if I left OK(_) empty it does nothing. Don't know if there's an nop function or similar